src/lib/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+
+type AvatarSizes = "extra large" | "large" | "medium" | "small" | "tiny";
+type AvatarStatus = "online" | "offline";
```

Comment:
I'm not sure about online/offline names. I think we could make it more generic like active/unactive. What do yu think?

Reply:
Makes sense to me, i will change it